recipes/fastrlock/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "fastrlock" %}
+{% set version = "0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6abdbb35205792e2d2a8c441aaa41a613d43ee2d88b3af4fd9735ae7a5f7db6b
+
+build:
+  number: 0
+  # We need to pass --with-cython to force cython to regenerate
+  script: "{{ PYTHON }} -m pip install . --global-option=\"--with-cython\" --no-deps --ignore-installed --no-cache-dir -vvv"
```

Comment:
I have to look at the logs to see if it is actually working. I just have a feeling that because we can, we should regenerate the cython code. That way we can keep upgrading to newer python versions (3,7) without depending on the tar ball files.